.gitignore

```diff
@@ -1,3 +1,2 @@
 node_modules
 npm-debug.log
-advisories.json
```

Comment:
rather than moving this line, add an exception for test/data/advisories.json by putting it in this file prefixed with an exclamation point

`!test/data/advisories.json`